src/style.css

```diff
@@ -1,3 +1,74 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  --hilight-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.logo-link {
+  display: flex;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+}
+
+.navigation {
+  display: flex;
+  align-items: center;
+  list-style: none;
+}
+
+.nav-link {
+  display: inline-block;
+  height: 60px;
+  line-height: 60px;
+  font-family: Roboto, sans-serif;
+  color: #000;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+li {
+  margin: 0 10px;
+}
```

Comment:
From the checklist: 'Don't use tag names for styling (except html and body)'. Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`.